src/routes.php

```diff
@@ -13,14 +13,14 @@
     'items/show' => ['ItemController', 'show', ['id']],
     'items/add' => ['ItemController', 'add',],
     'items/delete' => ['ItemController', 'delete',],
```

Comment:
I guess now you can remove item stuff ;)

Reply:
Done